alchemiscale_fah/compute/service.py

```diff
@@ -312,7 +312,9 @@ async def async_cycle(
                 self._refresh_cert_update_thread()
 
             done, pending = await asyncio.wait(
-                async_tasks, return_when=asyncio.FIRST_COMPLETED
+                async_tasks,
+                return_when=asyncio.FIRST_COMPLETED,
+                timeout=self.sleep_interval,
```

Comment:
This introduces a possible `TimeoutError` if a `Task` doesn't come in fast enough. Is this handled properly?